misc/verilator.mk.template

```diff
@@ -9,4 +9,4 @@ simulate: {testbench_exe_filepath} {testbench_inputs_filepath}
 	{testbench_exe_filepath} < {testbench_inputs_filepath}
 
 {testbench_exe_filepath}: {testbench_filepath}
-	$(VERILATOR) -o {testbench_exe_filepath} --assert --main --timing --Mdir {obj_dirpath} --cc --exe --build $^ {extra_verilator_args}
\ No newline at end of file
+	$(VERILATOR) --top-module testbench -o {testbench_exe_filepath} --assert --main --timing --Mdir {obj_dirpath} --cc --exe --build $^ {extra_verilator_args}
```

Comment:
Thanks for the change! Here's my suggestion on how to do this more cleanly:

1. Convert the above change to `--top-module {top_module}`
2. In [this call to format](https://github.com/uwsampl/lakeroad/blob/1ccfb09059e9fb145b943f18a43b72f5e21e9c7e/bin/simulate_with_verilator.py#L95-L105), add a `top_module="testbench"` argument. 

It's still hardcoded, but at least it makes the Makefile template more flexible.

If we wanted to make this even more flexible, we'd do the following:
1. Replace `module testbench` with `module {testbench_module_name}` in verilator_testbench.sv.template.
2. In the call to `format` which fills out the verilator_testbench.sv.template file, pass in the same module name that we pass to the `format` for the Makefile template (ideally stored in a variable somewhere, or taken as an argument to the script, perhaps an argument with the default value `testbench`).

If you feel like making a bigger PR, feel free to do the second set of instructions too. But just doing the first will be good enough!

Reply:
Awesome, thanks for the input. I'll start working on this change and update this PR